OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/CallGraphAnalysis.scala

```diff
@@ -121,25 +120,27 @@ class CallGraphAnalysis private[cg] (
 
     override final def processMethod(
         callContext: ContextType,
-        tacEP:       EPS[Method, TACAI]
+        tacEPOpt:    Option[EPS[Method, TACAI]]
     ): ProperPropertyComputationResult = {
-        val state = new CGState[ContextType](callContext, tacEP)
-        if (tacEP ne null)
+        if (tacEPOpt.isDefined) {
+            val state = new TACAIBasedCGState[ContextType](callContext, tacEPOpt.get)
             processMethod(state, new DirectCalls())
-        else
-            returnResult(new DirectCalls(), enforceCalleesResult = true)(state)
+        } else {
+            val state = new CGState[ContextType](callContext)
```

Comment:
Having two different kinds of state here looks strange. If the code below actually did anything (see other comment), I'd expect this to fail in the continuation, because the continuation would get an unexpected kind of state.

Reply:
Since there is no continuation (see https://github.com/opalj/opal/pull/220#discussion_r1766541607), it cannot fail. I will remove the state entirely (see the other comment again).